helsingborg-stad/app-mitt-helsingborg
# feat: add text for close dialog for form completions

## Explain the changes you’ve made
Make it possible to display another Close dialog text during completions (completions and random check).
Added new types for completions

## Explain why these changes are made
The old dialog message were wrong for completions

## Explain your solution
Added enum for which dialog template to use in the form, and created an object containing texts to display depending on current dialog template. 
Also remove formDialog since it was just a wrapper for CloseDialog.

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios`
3. Test running the application with a case that is NOT in a state for completion and check the close dialog text in the form.
4. ... or run the application with a case that is IN a state for completion and check the close dialog text

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Screenshots

**Close dialog during completion**
![simulator_screenshot_73048D18-5238-4658-9656-76D359E84561](https://user-images.githubusercontent.com/81250970/152349388-ad8fff5f-fc3c-4577-a44f-ec93da22924f.png)

**Close dialog normally**
![simulator_screenshot_E50F76EB-5A57-4C66-9150-28EEBEA31920](https://user-images.githubusercontent.com/81250970/152349521-25299ead-e6a8-4f83-82cf-1c499a9d34e2.png)
